src/main/java/ezvcard/io/json/JCardRawWriter.java

```diff
@@ -74,6 +77,23 @@ public JCardRawWriter(JsonGenerator generator) {
 		this.generator = generator;
 		this.closeGenerator = false;
 		this.wrapInArray = false;
+		ownGenerator = false;
+	}
+
+	/**
+	 * @deprecated Since 1.9.10 use {@link #isPrettyPrint}
+	 */
+	@Deprecated
```

Comment:
You can just remove the `setIndent` and `isIndent` methods.  Throughout the life time of this project, I have never made anything deprecated.  I understand the value of deprecation, but I would rather not have to deal with the cruft that deprecated methods add to the code base.  Also:  yee haw!